1-js/06-advanced-functions/02-rest-parameters-spread-operator/article.md

```diff
@@ -147,21 +147,21 @@ alert( Math.max(arr) ); // NaN
 */!*
 ```
 
```

Comment:
Please,

- The code above seems to contain an error conflict.
(In doubt, would you check the English website [article](https://javascript.info/rest-parameters-spread-operator) (link here)?)

Reply:
Same goes here.